src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final RegistrationServiceImpl REGISTRATION_SERVICE
+            = new RegistrationServiceImpl();
+    private User defaultCorrectUser;
+
+    @BeforeEach
+    void setUp() {
+        defaultCorrectUser = new User();
+        defaultCorrectUser.setAge(20);
+        defaultCorrectUser.setLogin("correct login");
+        defaultCorrectUser.setPassword("correctPassword");
+    }
+
+    @Test
+    void register_NullUser_NotOk() {

```

Comment:
```suggestion
    void register_nullUser_notOk() {
```

This style must be observed in all methods.